pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -861,7 +861,12 @@ func Convert_v1_Watchdog_To_api_Watchdog(source *v1.Watchdog, watchdog *api.Watc
 		watchdog.Model = "i6300esb"
 		watchdog.Action = string(source.I6300ESB.Action)
 		return nil
+	} else if source.Diag288 != nil {
+		watchdog.Model = "diag288"
+		watchdog.Action = string(source.Diag288.Action)
+		return nil
```

Comment:
I wonder why shouldn't we rework this and create an arch specific watchdog config?
Something like:
```
func (c converterS390X) ConvertWatchdog(source *v1.Watchdog, watchdog *api.Watchdog) error {
    if source.Diag288 != nil {
        watchdog.Model = "diag288"
        watchdog.Action = string(source.Diag288.Action)
        return nil
    }
    return fmt.Errorf("something")
}```

Reply:
Made necessary changes and added arch specific watchdog config.